app/src/main/kotlin/com/wire/android/ui/settings/SettingsScreen.kt

```diff
@@ -2,105 +2,34 @@ package com.wire.android.ui.settings
 import androidx.compose.ui.text.style.TextAlign
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
-import androidx.hilt.navigation.compose.hiltViewModel
-import com.wire.android.kaliumFileWriter
-import com.wire.android.ui.common.button.WireButtonState
-import com.wire.android.ui.common.textfield.WirePrimaryButton
-import com.wire.android.ui.theme.wireDimensions
-import com.wire.android.util.LOG_FILE_NAME
-import com.wire.android.util.startFileShareIntent
-import com.wire.kalium.logger.KaliumLogLevel
-import com.wire.kalium.logic.CoreLogger
-import java.io.File
-
 
 // the logic here is just temporary and will be updated to be added to the VM once we build the settings screen
 @Composable
 fun SettingsScreen() {
-    val settingsViewModel: SettingsViewModel = hiltViewModel()
-
-    val context = LocalContext.current
-
     Column(
-        modifier = Modifier
-            .fillMaxSize()
-            .wrapContentSize(Alignment.Center)
+        modifier = Modifier.fillMaxSize().wrapContentSize(Alignment.Center)
     ) {
-        val checkedState = remember { mutableStateOf(settingsViewModel.isLoggingEnabled()) }
-
-        Text(
-            text = "Enable logging",
-            modifier = Modifier
-                .fillMaxWidth()
-                .padding(MaterialTheme.wireDimensions.spacing16x)
-        )
-
-        Switch(
-            modifier = Modifier
-                .fillMaxWidth()
-                .padding(MaterialTheme.wireDimensions.spacing16x),
-            checked = checkedState.value,
-            onCheckedChange = {
-                checkedState.value = it
-                settingsViewModel.enableLogging(it)
-
-                if (it) {
-                    kaliumFileWriter.init(context.cacheDir.absolutePath)
-                    CoreLogger.setLoggingLevel(
-                        level = KaliumLogLevel.DEBUG, kaliumFileWriter
-                    )
-                } else {
-                    kaliumFileWriter.clearFileContent(
-                        File(context.cacheDir.absolutePath + "/logs/" + LOG_FILE_NAME)
-                    )
-                    CoreLogger.setLoggingLevel(
-                        level = KaliumLogLevel.DISABLED, kaliumFileWriter
-                    )
-                }
-            }
-        )
-
-        WirePrimaryButton(
-            state = if (checkedState.value) WireButtonState.Default else WireButtonState.Disabled,
-            text = "Share the log",
-            onClick = { context.startFileShareIntent(context.cacheDir.absolutePath + "/logs/" + LOG_FILE_NAME) },
-            fillMaxWidth = true,
-            modifier = Modifier
-                .fillMaxWidth()
-                .padding(MaterialTheme.wireDimensions.spacing16x)
-        )
-
-
         Text(
             text = "Settings is under construction",
             fontWeight = FontWeight.Bold,
             color = Color.Black,
```

Comment:
Can you also change this color to be for instance `MaterialTheme.colorScheme.onBackground`? 😃